src/Authentication/ITokenProvidersFactory.cs

```diff
@@ -6,5 +6,5 @@ namespace Microsoft.Artifacts.Authentication;
 
 public interface ITokenProvidersFactory
 {
-    Task<IEnumerable<ITokenProvider>> GetAsync(Uri authority);
+    Task<IEnumerable<ITokenProvider>> Get(Uri authority);
```

Comment:
>Get [](http://example.com/codeflow?start=38&length=3)

does this need to change? will be breaking if others update their package version

Reply:
would avoid any of these unless required for the public types